src/components/calcite-popover/resources.ts

```diff
@@ -13,3 +13,5 @@ export const TEXT = {
 export const POPOVER_REFERENCE = "data-calcite-popover-reference";
 export const ARIA_CONTROLS = "aria-controls";
 export const ARIA_EXPANDED = "aria-expanded";
+
+export type PopoverFocusId = "close-button";
```

Comment:
Can we drop the component from all focus ID type names? It's already implied by the module.

Reply:
@jcfranco it was like that but the typings were renaming one of them to Focus1.

Reply:
> the typings were renaming one of them to FocusId

I didn't follow that. Can you elaborate?

Reply:
Yeah, when I have two exported FocusId interfaces, it does this:

```

import { FocusId } from "./components/calcite-pick-list/shared-list-logic";
--
31 | import { ICON_TYPES } from "./components/calcite-pick-list/resources";
32 | import { FocusId as FocusId1 } from "./components/calcite-popover/calcite-popover";


```

https://unpkg.com/browse/@esri/calcite-components@1.0.0-next.122/dist/types/components.d.ts

Then it causes issues with using the Popover cus the typings are wrong.

Reply:
Updating based on our side conversation, this seems like a Stencil compiler issue. Looks like named imports do not get assigned properly for all cases. Renaming the imports is a good workaround for now. Once fixed, we can revisit simplifying the type name.